data/campaigns/Descent_Into_Darkness/utils/abilities.cfg

```diff
@@ -580,7 +580,7 @@
         [filter_self]
             [filter_adjacent]
                 count=1
-                type=Walking Corpse,Soulless
+                type_adv_tree=Walking Corpse
```

Comment:
Really not sure if Soulless should fall into the same usage as WC here in this ability. Like I expect a soulless corpse to deal more damage, while the ability only seems to want lvl 0 WC

Reply:
@knyghtmare are you saying that the existing code has a bug? Maybe it does, but I don't see an error in @newfrenchy83's changes, which AFAICS won't change the gameplay unless using an add-on.

If it is a bug, I feel it should be raised as a separate issue.

Reply:
What I meant was that the ability is scripted for just WCs. I meant is that if a Soulless is used, base damage should be higher and not "10"

Reply:
Why should the damage be higher? It's already a powerful attack, and using Soulless instead of WCs already has the advantage that the zombies themselves will have more hp remaining afterwards.

Reply:
@knyghtmare value of weapon special here was not subject of this PR, if it is a issue then open a issue in git but don't request a change here.